### PR TITLE
Update dev docs and rzup readme to use revamed rzup

### PR DESCRIPTION
**Pull request by SchmErik**

### CODE REVIEW

**Review by flaub — APPROVED** [review]
I don't know if this is complete, but looks good. @risc0/platform & @risc0/devrel let's take a pass of using these new instructions to ensure it works.

File: website/api/zkvm/install.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ For users who prefer manual installation, follow these steps:
 
 > At this time, there is only one release of the `cpp` toolchain. Installing the default version will fall back to this release.
 
-2. For x86-64 macOS, you must run `rzup toolchain rust build`.
+2. For x86-64 macOS or arm64 linux, you must build `rzup` from source. Clone the repository by running `git clone https://github.com/risc0/risc0.git`. Go to the root of the repository and run `cargo install --path rzup`. Build and install the rust toolchain by running `rzup toolchain build rust`.
```
Comment (flaub) [inline website/api/zkvm/install.md:44]:
```suggestion
2. For all other systems (e.g. macOS/x86-64, Linux/arm64), you must build `rzup` from source. 
* Clone the repository with: `git clone https://github.com/risc0/risc0.git`. 
* In the root of the repository, install `rzup` with: `cargo install --path rzup`.
* Build and install the rust toolchain with: `rzup toolchain build rust`.
```

Comment (SchmErik) [reply website/api/zkvm/install.md]: suggestion applied!

File: website/api/zkvm/install.md [hunk 2]
```diff
@@ -28,11 +28,14 @@ For a specific version, use `rzup install --version <version>`, where the `<vers
 
 See `rzup --help` for more options. You can find out more about `rzup` [here](https://github.com/risc0/risc0/tree/main/rzup).
 
+### Installation for all other systems (e.g. x86-64 macOS, arm64 Linux)
+For x86-64 macOS or arm64 linux, you must build `rzup` from source. Clone the repository by running `git clone https://github.com/risc0/risc0.git`. Go to the root of the repository and run `cargo install --path rzup`. Build and install the `cargo-risczero` by first checking out the branch `release-*` where `*` is `[major release number].[minor release number]` of your desired zkVM version. For example, if you would like to install version 1.1.0, run `git checkout origin/release-1.1` and run `cargo install --path risc0/cargo-risczero`. Build and install the rust toolchain by running `rzup toolchain build rust` this command may require utilities such as `cmake` and the `ninja` build system to be installed.
```
Comment (flaub) [inline website/api/zkvm/install.md]: This is hard to read, can we split this out into steps?

Comment (flaub) [reply website/api/zkvm/install.md]: I also think this should go down into Manual Installation

**Pull request merged**